README.md

```diff
@@ -1 +1,15 @@
-# DigitalBank
+
+Description:
+    This repository will focus on automated testing on the "Digital Bank" website
+
+Prerequisites:
+    - Install Java JDK 11
+    - Node.js v.12
+    - Install Docker to run the page to test
+    - Install Visual Studio Code
+    - Clone this repository : (https://github.com/digisic/digitalbank-gen-one)
+    
+Setup:
+    - Create a new folder where the project is going to be located
+    - In Docker, deploy the webpage so it can be tested
```

Comment:
Maybe provide more detail about specific commands they'll have to use to achieve this